docs/how-to/h-upgrade/h-upgrade-minor.md

```diff
@@ -110,9 +110,9 @@ Example with channel selection and juju 3.x:
 ```shell
 juju refresh postgresql-k8s --channel 14/edge --trust
 ```
-Example with specific revision selection:
+Example with specific revision selection (do NOT miss OCI resource!):
```

Comment:
There is a call to action inside the parenthesis, but the sentence is a heading style without action (verb). That looks a bit strange to me. Can we do the warning as an admonition after the code block?